plugins/dfu/dfu-device.c

```diff
@@ -65,6 +65,19 @@
  */
 #define	FU_QUIRKS_DFU_FORCE_VERSION		"DfuForceVersion"
 
+/**
+ * FU_QUIRKS_DFU_REMOVAL_DELAY:
+ * @key: the USB device ID, e.g. `USB\VID_0763&PID_2806`
+ * @value: the device removal delay, in ms, e.g. `30000`
+ *
+ * Specify a device removal delay for re-enumeration. This is required
+ * if it takes longer time than the default (FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE)
+ * for being detached to DFU or attached to normal mode.
+ *
+ * Since: 1.0.1
+ */
+#define	FU_QUIRKS_DFU_REMOVAL_DELAY		"DfuRemovalDelay"
```

Comment:
No sure the name "DfuRemovalDelay" is proper here. I just used a similar one with FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE